src/components/Editor.js

```diff
@@ -108,6 +108,22 @@ const Editor = React.createClass({
     shortcuts: PropTypes.object
   },
 
+  onKeyDown: function(e) {
+    let { key, target } = e;
+    let codeWrapper = this.editor.codeMirror.getWrapperElement();
+    let textArea = codeWrapper.querySelector("textArea");
+
+    if (key === "Escape" && target == textArea) {
+      e.stopPropagation();
+      e.preventDefault();
+      codeWrapper.focus();
+    } else if (key === "Enter" && target == codeWrapper) {
+      e.preventDefault();
+      // Focus into editor's text area
+      textArea.focus();
+    }
+  },
```

Comment:
hmm, i think this could be moved to [utils/editor](https://github.com/devtools-html/debugger.html/blob/master/src/utils/editor.js)

```js
function onKeyDown(codeMirror, e) {
  // stuff
}
```